tests/src/test/scala/com/nvidia/spark/rapids/StringFunctionSuite.scala

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2021, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nvidia.spark.rapids
+
+class StringFunctionSuite extends SparkQueryCompareTestSuite {
+  testSparkResultsAreEqual("Test repeat string(s) with scalar string and column repeatTimes",
+    intsDf) {
+    frame => frame.selectExpr(
+      "repeat(NULL, ints)",
+      "repeat('abc123', ints)"
```

Comment:
Are these Scala tests adding any value beyond what was tested in PySpark via pytest?  These are all going through the Spark APIs and I don't see how they're adding any value beyond what we could accomplish in PySpark.  Apologies if I'm missing something.

Reply:
Sorry maybe I'm wrong here when testing using Spark expresison. I'm rewriting the tests with `columnarEval` on `GpuStringRepeat` instance.